Naming/src/main/java/com/epam/engx/cleancode/naming/task1/CollectOrderService.java

```diff
@@ -1,28 +1,27 @@
 package com.epam.engx.cleancode.naming.task1;
 
-
 import com.epam.engx.cleancode.naming.task1.thirdpartyjar.CollectionService;
 import com.epam.engx.cleancode.naming.task1.thirdpartyjar.Message;
 import com.epam.engx.cleancode.naming.task1.thirdpartyjar.NotificationManager;
 import com.epam.engx.cleancode.naming.task1.thirdpartyjar.Order;
 
-public class CollectOrderService implements IOrderService {
+public class CollectOrderService implements OrderService {
 
-    private CollectionService ser1;
-    private NotificationManager ser2;
+    private CollectionService collectionService;
+    private NotificationManager notificationManager;
 
-    public void submitOrder(Order pOrder) {
-        if (ser1.isEligibleForCollection(pOrder))
-            ser2.notifyCustomer(Message.READY_FOR_COLLECT, 4); // 4 - info notification level
+    public void submit(Order order) {
+        if (collectionService.isEligibleForCollection(order))
+            notificationManager.notifyCustomer(Message.READY_FOR_COLLECT, 4); // 4 - info notification level
```

Comment:
The goal of every programmer should be to write code so clean and expressive that code comments are unnecessary. 

```suggestion
            notificationManager.notifyCustomer(Message.READY_FOR_COLLECT, INFO_NOTIFICATION_LEVEL); 
```